frontend/recorder/SaveScreen.tsx

```diff
@@ -84,14 +84,15 @@ export class _SaveScreen extends React.PureComponent<SaveScreenProps> {
 
         /* TODO: select target among user grants */
         return (
-            <form>
+            <form className="save-screen">
                 <FormGroup labelFor='eventsUrlInput' label={"URL évènements"}>
```

Comment:
This form has some non-i18n labels, and also :

```suggestion
                <FormGroup labelFor='eventsUrlInput' label={"URL événements"}>
```